test/smoke/src/areas/positron/plots/plots.test.ts

```diff
@@ -60,6 +61,129 @@ plt.show()`;
+h.edge("A", "B")
+h.edge("A", "C")
+
+IPython.display.display_png(h)`;
+
+				logger.log('Sending code to console');
+				await app.workbench.positronConsole.executeCode('Python', script, '>>>');
+
+				await app.workbench.positronPlots.waitForCurrentStaticPlot();
+
+				await app.workbench.positronPlots.clearPlots();
+
+				await app.workbench.positronPlots.waitForNoPlots();
+			});
+
+			it('Python - Verifies the plots pane action bar - Plot actions', async function () {
+				const app = this.app as Application;
+
+				const scriptPlot1 = `import graphviz as gv
+import IPython
+
+h = gv.Digraph(format="svg")
+names = [
+    "A",
+    "B",
+    "C",
+]
+
+# Specify edges
+h.edge("A", "B")
+h.edge("A", "C")
+
+IPython.display.display_png(h)`;
+
+				const scriptPlot2 = `import matplotlib.pyplot as plt
+
+# x axis values
+x = [1,2,3]
+# corresponding y axis values
+y = [2,4,1]
+
+# plotting the points
+plt.plot(x, y)
+
+# naming the x axis
+plt.xlabel('x - axis')
+# naming the y axis
+plt.ylabel('y - axis')
+
+# giving a title to my graph
+plt.title('My first graph!')
+
+# function to show the plot
+plt.show()`;
+				logger.log('Sending code to console');
+				const clearPlotsButton = app.workbench.positronPlots.clearPlotsButton;
+				const nextPlotButton = app.workbench.positronPlots.nextPlotButton;
+				const previousPlotButton = app.workbench.positronPlots.previousPlotButton;
+				const plotSizeButton = app.workbench.positronPlots.plotSizeButton;
+				const savePlotButton = app.workbench.positronPlots.savePlotButton;
+				const copyPlotButton = app.workbench.positronPlots.copyPlotButton;
+				const zoomPlotButton = app.workbench.positronPlots.zoomPlotButton;
+
+				// default plot pane state for action bar
+				await plotSizeButton.isNotVisible();
+				await savePlotButton.isNotVisible();
+				await copyPlotButton.isNotVisible();
+				await zoomPlotButton.isNotVisible();
+
+				// create plots separately so that the order is known
+				await app.workbench.positronConsole.executeCode('Python', scriptPlot1, '>>>');
+				await app.workbench.positronPlots.waitForCurrentStaticPlot();
+				await app.workbench.positronConsole.executeCode('Python', scriptPlot2, '>>>');
+				await app.workbench.positronPlots.waitForCurrentPlot();
+
+				expect(await clearPlotsButton.getAttribute('disabled'), 'Clear plots button should not be disabled').toBeUndefined();
+				expect(await nextPlotButton.getAttribute('disabled'), 'Next plot button should be disabled').toBeDefined();
+				expect(await previousPlotButton.getAttribute('disabled'), 'Previous plot button should not be disabled').toBeUndefined();
+				expect(await plotSizeButton.getAttribute('disabled'), 'Plot size button should not be disabled').toBeUndefined();
+				expect(await savePlotButton.getAttribute('disabled'), 'Save plot button should not be disabled').toBeUndefined();
+				expect(await copyPlotButton.getAttribute('disabled'), 'Copy plot button should not be disabled').toBeUndefined();
```

Comment:
You should talk to @sharon-wang , In one of her pending PRs, she added `isDisabled()` and `isEnabled()` to the baseElement. I think that would work in place of this. (same for the ones below).  But maybe I misunderstand the intent.

Reply:
Oh wow, that's way easier with what she's got!

Reply:
`expect(locator).not.toBeDisabled();`

This works pretty well! Using the `Locator` is the trick instead of working with the `BaseElement`. I think this entire test can use the `Locator` since it supports everything that's needed.